pkg/executor/brie.go

```diff
@@ -101,7 +100,7 @@ func (p *brieTaskProgress) Close() {
 	p.lock.Lock()
 	current := atomic.LoadInt64(&p.current)
 	if current < p.total {
-		p.cmd = fmt.Sprintf("%s Cacneled", p.cmd)
+		p.cmd = fmt.Sprintf("%s Cancelled", p.cmd)
```

Comment:
`Cancelled` or `Canceled`?

Reply:
If I recall right TiKV prefers AmE (canceled). For me I think both will be OK.